### ISSUE TITLE
Host flex.symfony.com could not resolved (from all world, browser too)

### ISSUE THREAD

**Issue by rizhenkov** [symfony/flex#909]
Hello all! 
I faced with this just now while trying to install th Symfony 4 project through `composer install`.
I got this output: 
```console
$ composer i
Installing dependencies from lock file (including require-dev)
Verifying lock file contents can be installed on current platform.
The following exception probably indicates you have misconfigured DNS resolver(s)

In CurlDownloader.php line 375:
                                                                                                                   
  curl error 6 while downloading https://flex.symfony.com/versions.json: 

Could not resolve host: flex.symfony.com  
```

And there is NOT MY LOCAL DNS problem, because I did additional step to check that:

1. I checked through side service https://dnschecker.org/#A/flex.symfony.com
And output shows that no place in internet where this domain is works: 

<details>
  <summary>Show screenshot</summary>
  
  <img width="1314" alt="image" src="https://user-images.githubusercontent.com/987517/166421609-35327691-063c-4df0-9262-d5d29ee33d1e.png">
  
</details>
2. I did addition steps and successfully check that my local DNS and cURL is working: 

<details>
  <summary>Show output</summary>
  
  ```console
$ php -i | grep cURL
cURL support => enabled
cURL Information => 7.74.0

$ ping google.com
PING google.com (142.250.203.142) 56(84) bytes of data.
64 bytes from waw07s06-in-f14.1e100.net (142.250.203.142): icmp_seq=1 ttl=37 time=10.9 ms

$ curl https://google.com
<HTML><HEAD><meta http-equiv="content-type" content="text/html;charset=utf-8">
<TITLE>301 Moved</TITLE></HEAD><BODY>
<H1>301 Moved</H1>
The document has moved
<A HREF="https://www.google.com/">here</A>.
</BODY></HTML>
```
  
</details>

So I look for some quick workaround to install that project and I think this DNS trouble on server need to be fixed too.

**Comment by nicolas-grekas** [comment]
You might have missed https://symfony.com/blog/the-old-flex-infrastructure-is-shutting-down

TL;DR: run `composer update symfony/flex --no-plugins --no-scripts`

**Comment by benkmugo** [comment]
Thanks @nicolas-grekas the link and note were helpful.

**Comment by hongdoan1** [comment]
Thank you, @nicolas-grekas
